src/styles/Blocks/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: var(--ColorWhite);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+  padding: 32px 16px 16px;
+}
+
+.card__logo {
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--ColorBigText);
+}
+
+.card__info {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--ColorSmallText);
+  margin-top: 4px;
+}
+
+.card__reviews {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__reviewsInfo {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  line-height: 18px;
+  padding-top: 24px;
+  padding-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  color: var(--ColorSmallText);
+}
+
+.card__price-count {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--ColorBigText);
+}
+
+.card__button {
+  border: 1px solid var(--ColorBlue);
+  background: var(--ColorBlue);
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+  font-weight: 700;
+  font-style: bold;
+  font-size: 14px;
+  line-height: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: var(--ColorWhite);
+  cursor: pointer;
+}
+
+.card__button:hover {
+  color: var(--ColorBlue);
+  background: var(--ColorWhite);
+  cursor: pointer;

```

Comment:
no need to set `cursor: pointer;` here if you already set it to the main state

Reply:
Thank you. Changed